test/common_nn.py

```diff
@@ -510,8 +521,8 @@ def multilabelmarginloss_reference(input, target, size_average=True, reduce=True
     ),
     dict(
         module_name='MultiLabelSoftMarginLoss',
-        input_size=(5, 10),
-        target_fn=lambda: torch.rand(5, 10).mul(2).floor(),
+        input_size=(10,),
+        target_fn=lambda: torch.rand(10).mul(2).floor(),
```

Comment:
What are these changes for?

Reply:
Oops, that shouldn't be there. good catch